final_assignment/alexandru_antochi/alexandru_antochi.py

```diff
@@ -0,0 +1,109 @@
+import json
+import os
+import subprocess
+import shutil
+from datetime import datetime
+import time
+import urllib
+
+fail = 0
+
+
+def parse_script():
+    """Functia citeste scriptul dat si il executa"""
+
+    log = open("tuxylog.log", 'a+')  # se creeaza fisierul de log
+    try:
+        with open('tuxy.json', encoding='utf-8') as file:
+            data = json.load(file)
+    except IOError:
+        print("Nu se poate deschide fisierul")
+        return
+    if 'config' in data:
+        for command in data['config']:
+            if command == "hostname":
+                os.system("hostname")
+                os.system(command)
+                log.write("Hostname set to: %s" % command)
+            if command == "user":
+                os.system("useradd")
+                log.write("User %s added." % data['config']['hostname']['users'][0])
+                os.system(data['config']['hostname']['users'][0])
+                for i in data['config']['hostname']['users'][0]:
+                    os.system(data['config']['hostname']['users'][0][i])  # scrie fiecare linie din script in shell
```

Comment:
Index-urile statice pot cauza probleme. Cred ca poate da si erori ca "index out of range" daca e gol container-ul.